src/styles/blocks/stars.scss

```diff
@@ -1,8 +1,11 @@
 .stars {
-  display: flex;
-  justify-content: space-between;
+  @extend %placeholder-flex;
   list-style: none;
 
+  &--4 {
```

Comment:
now modifier doesn't affect count of active stars, use n-th child to define that `4` means that there are four active stars, like you did in `stars` task
https://github.com/mate-academy/layout_stars/pull/491